src/libYARP_dev/src/devices/ServerGrabber/ServerGrabber.cpp

```diff
@@ -307,20 +307,19 @@ bool ServerGrabber::close() {
         delete poly;
         poly=nullptr;
     }
-    if(param.twoCameras)
+
+    if(responder2)
     {
-        if(responder2)
-        {
-            delete responder2;
-            responder=nullptr;
-        }
-        if(isSubdeviceOwned && poly2)
-        {
-            poly2->close();
-            delete poly2;
-            poly2=nullptr;
-        }
+        delete responder2;
+        responder=nullptr;
```

Comment:
Hi @barbalberto , just an uninformed comment: should it be `responder2=nullptr;` instead of `responder=nullptr;`?

Reply:
Ops, you are right. Good catch, thank you